ui/helpers/utils/permission.js

```diff
@@ -142,113 +137,81 @@ export const PERMISSION_DESCRIPTIONS = deepFreeze({
-            <span
-              className="tooltip-label-item"
-              key={`description-${path.join('/')}`}
+            <Text
+              as="span"
+              className="permission-label-item"
+              key={path.join('/')}
+              fontWeight={FontWeight.Medium}
             >
               {friendlyName}
-            </span>,
-            curve,
+            </Text>,
+            path.join('/'),
           ]),
+          description: t('permission_manageBip44AndBip32KeysDescription'),
         };
       }
 
       return {
         ...baseDescription,
         label: t('permission_manageBip32Keys', [
-          <span className="permission-label-item" key={path.join('/')}>
-            {path.join('/')}
-          </span>,
-          curve,
-        ]),
-        description: t('permission_manageBip32KeysDescription', [
-          <span
-            className="tooltip-label-item"
-            key={`description-${path.join('/')}`}
+          <Text
+            as="span"
+            fontWeight={FontWeight.Medium}
+            className="permission-label-item"
+            key={path.join('/')}
           >
-            {path.join('/')}
-          </span>,
-          curve,
+            {`${t('unknownNetworkForKeyEntropy')} ${path.join('/')} (${curve})`}
+          </Text>,
         ]),
+        description: t('permission_manageBip44AndBip32KeysDescription'),
       };
     }),
-  [RestrictedMethods.snap_getBip44Entropy]: ({
-    t,
-    permissionValue,
-    targetSubjectMetadata,
-  }) =>
+  [RestrictedMethods.snap_getBip44Entropy]: ({ t, permissionValue }) =>
     permissionValue.caveats[0].value.map(({ coinType }, i) => ({
       label: t('permission_manageBip44Keys', [
-        <span className="permission-label-item" key={`coin-type-${coinType}`}>
-          {coinTypeToProtocolName(coinType) ||
-            t('unrecognizedProtocol', [coinType])}
-        </span>,
-      ]),
-      description: t('permission_manageBip44KeysDescription', [
-        <span
-          className="tooltip-label-item"
-          key={`description-coin-type-${coinType}`}
-        >
-          {coinTypeToProtocolName(coinType) ||
-            t('unrecognizedProtocol', [coinType])}
-        </span>,
-      ]),
-      leftIcon: IconName.Key,
-      weight: 1,
-      id: `key-access-bip44-${coinType}-${i}`,
-      message: t('snapInstallWarningKeyAccess', [
         <Text
-          key="1"
-          color={Color.primaryDefault}
-          fontWeight={FontWeight.Medium}
           as="span"
+          fontWeight={FontWeight.Medium}
+          className="permission-label-item"
+          key={`coin-type-${coinType}`}
         >
-          {getSnapName(targetSubjectMetadata?.origin, targetSubjectMetadata)}
-        </Text>,
-        <Text as="span" key="2" fontWeight={FontWeight.Medium}>
           {coinTypeToProtocolName(coinType) ||
-            t('unrecognizedProtocol', [coinType])}
+            t('unrecognizedProtocol', [
```

Comment:
Is there a reason not to use `unknownNetworkForKeyEntropy` for both BIP32 and BIP44?

Reply:
No reason. I forgot that one exists because of changing this later. Good catch. Refactored now.